app/react/Documents/components/DocumentSidePanel.js

```diff
@@ -39,6 +40,20 @@ export class DocumentSidePanel extends Component {
     }
   }
 
+  setDefaultDocumentToC(isEntity, documents, language) {
+    let defaultDocumentToC = this.props.file.toc;
+
+    if (!isEntity) {
+      const defaultDocument = {
+        ...entityDefaultDocument(documents, language, 'en'),
```

Comment:
This is not correct: "en" is not the default language.  The default language is collection-dependent and it comes from the DB from settings -> languages -> the one where default === true.

It is probably already in the store somewhere, as a lot of things, including the language you are redirected to upon landing on any page, depend on this.

Reply:
state.locale holds the 'current language' that the person has either selected or has been assigned automatically.  The entityDefaultLanguage expects three arguments: documents, entityLanguage and defaultLanguage.  EntityLanguage.

Since we have one entity per UI language, entityLanguage and stat.locacle are 100% of the time going to be the same value.  What it is expecting in the 3rd argument is the COLLECTION's default so that it can, if there is no match to the entity's language, try to pick the collection's default.

So you need to fetch it from state.collection (I'm almost sure this is always present) and, from languages, extract the one that has default: true.  That should be done in the mapStateToProps and you pass the language so that we prevent unnecessary rerenders.